client/app/domain/domain-validator.service.js

```diff
@@ -303,16 +303,16 @@ angular.module("services").service(
             case "DYNHOST":
                 return this.Validator.isValidIpv4(target);
```

Comment:
`this.Validator.isValidIpv4(target)` no longer exists, should use `this.Validator.ip.isValid(target, { version: 4 });`